Source/Management.Web/metrics/Metrics.tsx

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import { CommandBar, DetailsList, IColumn, ICommandBarItemProps, Stack } from '@fluentui/react';
+import React, { useEffect, useState } from 'react';
+import { Prometheus } from '../prometheus/Prometheus';
```

Comment:
I see you've copied a lot of code from a library. Is there no other parser of the Prometheus exposition format that would work?

Reply:
I searched around for it and couldn’t really find a good maintained library that wasn’t for Node. Not saying there isn’t one, but stumbled upon this code and just adapted it a little bit for our needs. Happy to use something completely different.